scrapers/GIGA.yml

```diff
@@ -28,6 +28,8 @@ xPathScrapers:
     common:
       $videoItem: //*[@id="list"]
     scene:
+      Code:
+        selector: $videoItem/ul/li[2]/ul/li[1]/dl[1]/dd[1]
```

Comment:
`$videoItem/ul/li/ul[1]/li[1]/dl[1]/dd`

seems to give the most reliable output

Reply:
Thanks and pushed change.